VirtoCommerce.Platform.Data.Assets.FileSystem/FileSystemBlobProvider.cs

```diff
@@ -141,70 +147,77 @@ public virtual BlobSearchResult Search(string folderUrl, string keyword)
                     Url = GetAbsoluteUrlFromPath(file),
                     ContentType = MimeTypeResolver.ResolveContentType(fileInfo.Name),
                     Size = fileInfo.Length,
-                    FileName = fileInfo.Name,
+                    Name = fileInfo.Name,
```

Comment:
Need use AbstractTypeFactory for concrete type instances creation instead of new operator